addons/voice/org.openhab.voice.gtts/pom.xml

```diff
@@ -0,0 +1,17 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+
+    <parent>
+        <artifactId>pom</artifactId>
+        <groupId>org.openhab.voice</groupId>
+        <version>2.4.0-SNAPSHOT</version>
+    </parent>
+
+    <artifactId>org.openhab.voice.gtts</artifactId>
```

Comment:
Do you think the package name `org.openhab.voice.gtts` is self explaining enough @kaikreuzer?

It may become confusing when another TTS engine is added that starts with a G or when Google creates a new TTS engine one day.

Using `org.openhab.voice.googlecloudtts` would for instance could solve that but it may become be a bit too long. ;-)

Reply:
What do you say for a gctts? I haven't changed this until we decide...

Reply:
How about googletts? I doubt that they will ever over another tts service, so this name should be unique enough and still immediately tells what it is about.

Reply:
Changed to googletts